tests/unit/unit_t_db_base.py

```diff
@@ -256,3 +257,44 @@ def create_search_index(self):
             }
         }
         self.search_ddoc.save()
+
+    def load_security_document_data(self):
+        """
+        Create a security document in the specified database and assign
+        attributes to be used during unit tests
+        """
+        self.sdoc = {
+            'admins': {'names': ['foo'], 'roles': ['admins']},
+            'members': {'names': ['foo1', 'foo2'], 'roles': ['developers']}
+        }
+        self.mod_sdoc = {
+            'admins': {'names': ['bar'], 'roles': ['admins']},
+            'members': {'names': ['bar1', 'bar2'], 'roles': ['developers']}
+        }
+        if os.environ.get('RUN_CLOUDANT_TESTS') is not None:
+            self.sdoc = {
+                'cloudant': {
+                    'foo1': ['_reader', '_writer'],
+                    'foo2': ['_reader']
+                }
+            }
+            self.mod_sdoc = {
+                'cloudant': {
+                    'bar1': ['_reader', '_writer'],
+                    'bar2': ['_reader']
+                }
+            }
+        if os.environ.get('ADMIN_PARTY') == 'true':
```

Comment:
Yep that certainly is why.  I just fixed the Jenkinsfile to remove the ADMIN_PARTY export since it is not needed.  This fixes #227 as well.  Done in https://github.com/cloudant/python-cloudant/pull/236/commits/ad78e222587c2cb84dc78e727190fd359484b7dd